### DISCUSSION TITLE
Allow custom middleware to be registered

### DISCUSSION THREAD

**Discussion by Larsklopstra** [filamentphp/filament#171] [Help] [answered]
Title explains itself.

Ideally set in `config/filament.php`

and registered at `src/Providers/RouteServiceProvider.php`

eg:

```php
<?php

return [
    'middleware' => [CheckIfDomainExists::class, CheckIfUserBelongsToDomain::class],
]
```

**Comment by Wulfheart** [comment, marked as answer]
For future reference: As of version 2 this is already implemented.
